src/main/scala/io/iohk/ethereum/db/storage/BlockHeadersStorage.scala

```diff
@@ -31,7 +29,7 @@ class BlockHeadersStorage(val dataSource: DataSource)
     blockHeader => compactPickledBytes(Pickle.intoBytes(blockHeader))
 
   override def valueDeserializer: IndexedSeq[Byte] => BlockHeader =
-    bytes => Unpickle[BlockHeader].fromBytes(ByteBuffer.wrap(bytes.toArray[Byte]))
+    byteSequenceToBuffer _ andThen Unpickle[BlockHeader].fromBytes
```

Comment:
WDYT about creating helper for `byteSequenceToBuffer _ andThen Unpickle[Type].fromBytes` ?

Reply:
I'd prefer to merge it ASAP, but added a TODO